draft-ietf-tls-exported-authenticator.md

```diff
@@ -479,22 +487,28 @@ When validating an authenticator, constant-time comparison should be used.
 ## Update of the TLS ExtensionType Registry
 
 IANA is requested to update the entry for server_name(0) in the registry for
-ExtensionType (defined in {{!TLS13}}) by replacing the value in the "TLS 1.3"
+ExtensionType (defined in {{!RFC8446}}) by replacing the value in the "TLS 1.3"
 column with the value "CH, EE, CR" and adding this document in the "Reference" column.
 
+The addition of "CR" to this column does not authorize the use of this extension
```

Comment:
@grittygrease I think we split the difference between adding a column and just having the text in the I-D but making sure the text is in the registry by asking IANA to add a note with text like:

      IANA is also requested to add the following note to the registry:
 
      The addition of the "CR" to the "TLS 1.3" column for the server_name(0) extension does not ...